Assets/Scripts/Tools/LabelVisibilityTool.cs

```diff
@@ -7,7 +7,7 @@ internal class LabelVisibilityTool : ITool
     readonly GraphService _graphService;
     readonly ToolPanelController _toolPanelController;
 
-    public string ToolName => "Label visibility";
+    public string ToolName => "Label\nvisibility";
```

Comment:
Code should not be the one handling text appearance. The UI element should have a max width and text wrapping set.

Reply:
i couldn't change it in prefab

Reply:
Why not? Was there a limitation?

Reply:
We have a TextMesh, and even with rect transform it does not provide text wrap (width & height values doesn't change anything)
"Basically there are no way to get an abitrary TextMesh's size without rendering it."
you can write whole class, that fits text to width, but I think in this case it would be overengineering

Reply:
wellthatsucks